src/flowMC/resource_strategy_bundle/RQSpline_GRW_PT.py

```
@@ -51,22 +53,34 @@ def __init__(
         n_training_loops: int,
         n_production_loops: int,
         n_epochs: int,
+        # --- Local sampler ---
         grw_step_size: Float | Float[Array, " n_dim"] = 1e-1,
```

Ruff (0.15.2)

[error] Syntax error in forward annotation: Expected an expression

(F722)